src/extent_dss.c

```diff
@@ -65,37 +67,41 @@ extent_dss_prec_set(dss_prec_t dss_prec) {
 	return false;
 }
 
+static void
+extent_dss_extending_start(void) {
+	spin_t spinner = SPIN_INITIALIZER;
```

Comment:
There were bootstrapping issues in this code prior to switching away from mutexes.